js/site.js

```diff
@@ -816,3 +793,34 @@ function iOSversion() {
     return parseInt(v[1], 10);
   }
 }
+
+/*
+Transforms the cities hash to be a single-level hash with name-okres-kraj as key
+*/
+function getTransformedCities() {
```

Comment:
Tie mesta tu trochu masirujem, lebo som vam nechcel rozbit cities.js (zda sa, ze to generuje skript?)